azurerm/resource_arm_automation_credential.go

```diff
@@ -75,13 +88,20 @@ func resourceArmAutomationCredentialCreateUpdate(d *schema.ResourceData, meta in
 
 	name := d.Get("name").(string)
 	resGroup := d.Get("resource_group_name").(string)
-	accName := d.Get("account_name").(string)
+	//CustomizeDiff should ensure one of these two is set
```

Comment:
as customize diff is broken (and we are working on a better way to achieve this with a new validator) i think we can remove this
```suggestion
```